lib/flat_search/olx_scraper.ex

```diff
@@ -65,12 +64,24 @@ defmodule FlatSearch.OlxScraper do
       negotiation: negotiable?(document),
       surface: get_surface(document),
       photo_links: get_photo_links(document),
-      link: url
+      link: url,
+      region: document |> get_localization() |> Enum.at(0) |> insensitive_string(),
+      city: document |> get_localization() |> Enum.at(1) |> insensitive_string(),
+      district: document |> get_localization() |> Enum.at(2) |> insensitive_string()
```

Comment:
how about moving this pipeline to another function?
`document |> get_localization() |> Enum.at(0) |> insensitive_string()`

taking the two args: ie. document and depth

Reply:
yeah, I was fighting with myself if I should do it or not